stable/prometheus-operator/templates/exporters/prometheus-operator/cleanup-crds.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: prometheus-operator-cleanup-crds
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-delete
```

Comment:
Wouldn't running this before deletion conflict with the resources still in the cluster?